jenkins_test.go

```diff
@@ -55,14 +55,29 @@ func TestCreateNodes(t *testing.T) {
 	assert.Equal(t, id3, node3.GetName())
 }
 
+func TestDeleteNodes(t *testing.T) {
```

Comment:
This test case will affect others. You can create an additional node `node4` in `TestCreateNodes()`, then delete it in this case.